src/contain/text.js

```diff
@@ -88,7 +88,7 @@ function getPlainTextRect(text, font, textAlign, textVerticalAlign, textPadding,
     return rect;
 }
 
-function getRichTextRect(text, font, textAlign, textVerticalAlign, textPadding, textLineHeight, rich, truncate) {
+function getRichTextRect(text, font, textAlign, textVerticalAlign, textPadding, rich, textLineHeight, truncate) {
```

Comment:
Since we need to pass so many parameters,
 I think it's better to use destructuring assignment syntax in es6, just like this.
```js
function getRichTextRect({text, font, textAlign, textVerticalAlign, textPadding, rich, textLineHeight, truncate}) {
     // main body
}
```
then we do not need to consider the order of parameters.